kong/llm/drivers/shared.lua

```diff
@@ -117,57 +165,67 @@ function _M.to_ollama(request_table, model)
 end
 
 function _M.from_ollama(response_string, model_info, route_type)
+  local output, _, analytics
```

Comment:
I saw in `kong/llm/drivers/cohere.lua`, a similar return value like `analytics` is called `metadata`, shall we rename it to metadata as well for consistency?

Reply:
Yes good idea

Fixed in 0cdecea6fd3e7ec378e2805f3807295078998f09